replace-all-dots.js

```diff
@@ -0,0 +1,4 @@
+var replaceDots = function (str) {
+
+    return str.replace(/\./g, '-');
+}
```

Comment:
Missing the **README.md file with the description of the problem domain from Code Wars, a link to the challenge on Code Wars, and an overview of your process for solving the challenge.**